tests/unit/components/FsExample.test.js

```diff
@@ -61,12 +61,12 @@ describe('FsExample.vue', () => {
     test('Render default contents', () => {
       const wrapper = shared.renderDefaultContents();
 
-      expect(wrapper)
+      expect(wrapper.html())
```

Comment:
To eliminate variables that could interfere with tests running, I'm converting these all to strings and not using a serializer, so Jest will just do a basic string comparison. (for now)